packages/ilios-common/addon/components/dashboard/courses-calendar-filter.js

```diff
@@ -95,9 +95,11 @@ export default class DashboardCoursesCalendarFilterComponent extends Component {
 
   @action
   scrollToLastYear(element, [year]) {
-    if (year === this.academicYear - 1) {
-      scrollIntoView(element.parentElement.parentElement, {
-        align: { top: 0 },
+    if (year === this.defaultExpandedYear) {
+      const { offsetTop } = element;
+      element.parentElement.scrollTo({
+        top: offsetTop,
+        behavior: 'instant',
```

Comment:
Is the screen meant to actually "jump-scroll"? Because it doesn't do any scrolling at all now.

Reply:
Correct and great question. The UI doesn't scroll, instead what happens is the most current academic year is focused at the top of the filter box. This is hard to test. What I did was create courses in several future years, then you will see that those are above the current year in the view and you have to scroll up to reach them. The UI doesn't move around though, it just starts in the middle.